modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/users/admin/management/toolbar/FilterContributorRegistryUtil.java

```diff
@@ -13,18 +13,16 @@
 
 import java.util.List;
 
+import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Drew Brokke
  */
-@Component(service = FilterContributorRegistry.class)
-public class FilterContributorRegistry {
+public class FilterContributorRegistryUtil {
```

Comment:
Hey @GabrielHenriquedev, please change the commit message, so it matches your changes, thanks.